tests/pf/test_mag_inversion_linear.py

```diff
@@ -115,28 +115,42 @@ def setUp(self):
         betaest = directives.BetaEstimate_ByEig()
 
         # Here is where the norms are applied
-        IRLS = directives.Update_IRLS(f_min_change=1e-4, minGNiter=1)
+        IRLS = directives.UpdateIRLS(f_min_change=1e-4)
         update_Jacobi = directives.UpdatePreconditioner()
         sensitivity_weights = directives.UpdateSensitivityWeights(every_iteration=False)
         self.inv = inversion.BaseInversion(
-            invProb, directiveList=[IRLS, sensitivity_weights, betaest, update_Jacobi]
+            invProb,
+            directiveList=[
+                IRLS,
+                sensitivity_weights,
+                betaest,
+                update_Jacobi,
+            ],
         )
 
     def test_mag_inverse(self):
         # Run the inversion
         mrec = self.inv.run(self.model)
         residual = np.linalg.norm(mrec - self.model) / np.linalg.norm(self.model)
 
+        # import matplotlib.pyplot as plt
         # plt.figure()
         # ax = plt.subplot(1, 2, 1)
-        # midx = int(self.mesh.shape_cells[0]/2)
-        # self.mesh.plot_slice(self.actvMap*mrec, ax=ax, normal='Y', ind=midx,
-        #                grid=True, clim=(0, 0.02))
-
+        # midx = int(self.mesh.shape_cells[0] / 2)
+        # self.mesh.plot_slice(
+        #     self.actvMap * mrec, ax=ax, normal="Y", ind=midx, grid=True, clim=(0, 0.02)
+        # )
+        #
         # ax = plt.subplot(1, 2, 2)
-        # midx = int(self.mesh.shape_cells[0]/2)
-        # self.mesh.plot_slice(self.actvMap*self.model, ax=ax, normal='Y', ind=midx,
-        #                grid=True, clim=(0, 0.02))
+        # midx = int(self.mesh.shape_cells[0] / 2)
+        # self.mesh.plot_slice(
+        #     self.actvMap * self.model,
+        #     ax=ax,
+        #     normal="Y",
+        #     ind=midx,
+        #     grid=True,
+        #     clim=(0, 0.02),
+        # )
```

Comment:
Is there a good reason to keep these commented lines? They are not run while running tests and therefore we are not keeping track that they are currently working as expected. What do you think?

Reply:
Yea for sure, I just kept them around because it is helpful to check the model when test fails. I am easy either way.

Reply:
I see. What if we add these plots as another test function/method that we decorate with `@pytest.mark.skip`? This way we won't be execute them when running the tests, but at least the code is not going to be commented out.

What do you think?

Reply:
Done